src/components/global-view-core.js

```diff
@@ -388,28 +404,41 @@ const GlobalViewCore = ({ timestampMin, timestampMax, timestampStep }) => {
                 aria-labelledby="modal-modal-title"
                 aria-describedby="modal-modal-description"
             >
-                <Box sx={modalStyle}>
-                    <Typography
-                        id="modal-modal-title"
-                        variant="h6"
-                        component="h2"
+                <Box sx={modalNoOverflowStyle}>
+                    <Box
+                        style={{
+                            position: 'sticky',
+                            zIndex: '1',
+                            paddingLeft: '32px',
+                            paddingTop: '15px',
+                            marginRight: '10px',
+                            backgroundColor: 'inherit',
+                        }}
                     >
-                        <FormattedMessage id="events" />
-                        <Button
-                            style={{ float: 'right' }}
-                            onClick={handleEventClose}
+                        <Typography
+                            id="modal-modal-title"
+                            variant="h6"
+                            component="h2"
                         >
-                            <Close />
-                        </Button>
-                    </Typography>
-                    {isLoadingEvent ? (
-                        <LinearProgress />
-                    ) : (
-                        <EventsTable
-                            id="modal-modal-description"
-                            taskData={getEventsData()}
-                        />
-                    )}
+                            <FormattedMessage id="events" />
+                            <Button
+                                style={{ float: 'right' }}
+                                onClick={handleEventClose}
+                            >
+                                <Close />
+                            </Button>
+                        </Typography>
+                    </Box>
+                    <Box sx={modalStyle}>
```

Comment:
You don't need to reapply all the style here, just add the overflow in auto and the maxHeight to scroll on the EventTable.